wiki/Multiplayer/en.md

```diff
@@ -1,451 +1,419 @@
+
 From Main menu,
-- Press "**Play**" then "**Multi**"; or
-- Press "**P**" then "**M**". (from Keyboard)
 
-You will be directed to the "**lobby**", the core of the Multi-play.
+- Press `Play` then `Multi`; or
+- Press `P` then `M`. (from Keyboard)
+
+If the connection request with _Bancho_ was successful, _osu!_ will be directed to the **lobby**, the core of the Multi-play.
 
 ### Notes
-- Keep in mind that downloading times can range from a few seconds to 15mins depending on your internet connection.
-- Losing connection to Bancho will send you back to Main Menu until connection is recovered to Bancho. This will force you to be considered "Quit" when playing a match.
-- "Quit" when playing a match.
-- Requires [osu!supporter][osu!supporter wikilink] in cuttingedge build.
+
+Keep in mind that downloading times can range from a few seconds to 15 minutes onwards depending on internet connection.
+
+Losing connection to _Bancho_ will direct _osu!_ back to Main Menu until connection to _Bancho_ has been recovered.
+When playing a match, a "Quit" will be given on the playerbox.
+
+If _not_ in _Stable_ build, the logged _osu!account_ must has an active _[osu!supporter][osu!supporter wikilink]_ to enter Multi.
 
 ## Lobby
+
 ![Example of Multiplayer Lobby][Multi_lobby image]
 
-This is a (waiting) Multiplayer Lobby.
--   Here is a list of things that a player can do in the lobby:-
-	-   Press one of the tabs above to sort out the game style they want.
-	-   Selecting the few options below the tab to sort out the list.
-	-   Creating a new "room" by pressing "New Game"
-	-   Press "Quick Join" to randomly join a room based on the beatmap you have.
-	-   Press one of the "room" to enter that "room" group.
-	-   Talk with other players using the chat console which are opened by default. Please be at your best behavior at all time.
-		-   You will be connected to \#lobby by default.
-	-   Check out the players logged into lobby by looking at the right of the screen.
-	-   Select and playing your jukebox songs at the upper-right of the screen.
+_This is a (waiting) Multiplayer Lobby._
+
+It will show all the available Match Setup in the _Bancho_ server.
+Sort out by game mode or fine-tune the search by filtering or searching for a player's name.
+[Chat Console][Chat Console wikilink] will be available at the screen and connected to \#lobby by default.
+
+Each row contains a Match Setup with its game mode symbol, game mode name (Team Mode), number of players inside/number of players allowed, rank difference from highest to lowest at the left section; large host profile image and smaller players profile image in the Match Setup along with the Match Setup name and the beatmap's name \[difficulty\] to be played for this match at the right section.
+
+Click on the row to enter the Match Setup.
+If the game mode symbol contains a lock, clicking on it will prompt a password request; fill in the Match Setup password correctly to enter the locked Match Setup.
```

Comment:
`If the game mode symbol contains a lock icon, clicking on it will prompt you to enter the room password; by entering the correct password, you will enter the locked Match Setup.`
I used "room", but idk if `room` and `Match Setup` are the same